app/src/main/java/org/mate/ui/EnvironmentManager.java

```diff
@@ -19,6 +19,7 @@
     public static final String ACTIVITY_UNKNOWN = "unknown";
     private static final String DEFAULT_SERVER_IP = "10.0.2.2";
     private static final int DEFAULT_PORT = 12345;
+    //private static final String DEFAULT_SERVER_IP = "localhost";
```

Comment:
Should not be added to the commit but kept locally.

Reply:
done